tests/test_interfaces.py

```diff
@@ -0,0 +1,595 @@
+# pylint: skip-file
+""" This module tests the interface module """
```

Comment:
I don't understand what you mean. I just check that the code calls `PyRIC`'s function and in some cases change the return values for some functions.